cogs/audio.py

```diff
@@ -263,6 +271,63 @@ def __init__(self, bot):
         else:
             await self.bot.say("There are no valid sound effects in the `data/audio/sfx` folder.")
 
+    @commands.command(pass_context=True, no_pm=True)
+    async def tts(self, ctx, lang: str, *, text: str="Please try again."):
+        """Convert your text to speech (Google TTS).
+        """
+        
+        text = self.chop(ctx, text)
```

Comment:
`!tts en my string`
text is "my string"
from the looks of it, chop will return: "ring"